### ISSUE TITLE
SR8 - Screen reader picks up some glyphicons

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#198]
When going through Services by Category with a screen reader, two of the glyphicons are picked up by the screen reader while the rest are skipped. Specifically, the icons under 'Computer, Printer and Server Support' as well as 'Research Support' are interpreted by NVDA. I believe this [fails criterion 1.1.1](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F39) under the WCAG 2.0.

"☁
link
Computer, Printer and Server Support
+
link
Research Support"
https://aoda.web.uoguelph.ca/aodademo/services
